pom.xml

```diff
@@ -98,6 +98,7 @@
         <org.jdom.version>1.1.3</org.jdom.version>
         <org.kohsuke.github-api.version>1.73</org.kohsuke.github-api.version>
         <org.mod4j.org.eclipse.core.expressions.version>3.4.100</org.mod4j.org.eclipse.core.expressions.version>
+        <org.postgresql.version>9.4-1201-jdbc41</org.postgresql.version>
```

Comment:
postgresql version shouldn't be at least the same than in the docker image ? 9.6 ?
https://github.com/eclipse/che/blob/che-multiuser/dockerfiles/cli/images.template#L6

Reply:
This one works fine for us since we use 9.5 for a long time. 
We will move to 9.6 as soon as https://dev.eclipse.org/ipzilla/show_bug.cgi?id=13161 will be approved. AFAIK we don't use any version specific features.

Reply:
@skabashnyuk yes it was not a negative comment from myself. Just that it's better to upgrade the versions to their same versioning scheme. (And I also know that CQs can take a looooong time ;-)

Reply:
Your comment was useful.  It remind me to do this https://github.com/eclipse/che/pull/6614